Fix build failure when running lint

major0 opened the pull request:
Commit 4744d40f introduced a trailing semicolon that violates the build
checks performed during `npm run lint`.

TobiTenno commented:
Looks like it's failing for other reasons rn. I've reached out to @Kaptard on discord. here's hoping he has a chance to grab that.
also, something you may want to consider doing: [commit message format](https://github.com/semantic-release/semantic-release#commit-message-format)

major0 commented:
> Looks like it's failing for other reasons rn. I've reached out to @Kaptard on discord. here's hoping he has a chance to grab that.
> also, something you may want to consider doing: [commit message format](https://github.com/semantic-release/semantic-release#commit-message-format)

Maybe.  I based my patch on the following:
```
$ git co development 
Switched to branch 'development'
Your branch is up to date with 'upstream/development'.
$ npm run lint

> warframe-items@0.0.0-development lint /home/mark.ferrell/Personal/warframe-items.git
> eslint --ignore-path .gitignore .


/home/mark.ferrell/Personal/warframe-items.git/build/wikia/transformers/transformWeapon.js
  141:69  error  Extra semicolon  semi

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

(node:9006) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! warframe-items@0.0.0-development lint: `eslint --ignore-path .gitignore .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the warframe-items@0.0.0-development lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mark.ferrell/.npm/_logs/2019-12-30T16_46_18_387Z-debug.log
```

```
$ git co bugfix/lint_failure 
Switched to branch 'bugfix/lint_failure'
Your branch is ahead of 'upstream/development' by 1 commit.
  (use "git push" to publish your local commits)

$ npm run lint

> warframe-items@0.0.0-development lint /home/mark.ferrell/Personal/warframe-items.git
> eslint --ignore-path .gitignore .

(node:9079) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
```

TobiTenno commented:
Your reason for the pr change is fine, I'm referring to it failing to fetch data from it's own local system in the failure on your branch.

TobiTenno commented:
@major0 could you push an empty commit so it retriggers the CI build? it looks like kaptard got that fixed
